### PR TITLE
Add Beta 1 changelog.

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

File: docs/changelog/1_0_b1.rst [hunk 1]
```diff
@@ -0,0 +1,332 @@
+==========
+1.0 Beta 1
+==========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 beta 1 "Sirius".
```
Comment (1st1) [inline docs/changelog/1_0_b1.rst:6]: Maybe we should call this Sirius A. Then the second beta or the RC can be called Sirius B. Thoughts?

File: docs/changelog/1_0_b1.rst [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+
+
+Migrations
+==========
+
+One of the major improvements in Beta release is the migrations
+support (`RFC 1000 <migrations_>`_). It is now possible to manage
```
Comment (1st1) [inline docs/changelog/1_0_b1.rst:13]: This reads strange, since migrations have been there for a while. I'd rephrase to something along the "This Beta release features the fully implemented RFC 1000 for the first time"

**Review by 1st1 — COMMENTED** [review]
We'll also need to add details about the new ports, but that can be done after this is merged (and when the ports finally land)

File: docs/changelog/1_0_b1.rst (hunk 1 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_b1.rst:6]: That would be kinda neat, but only if we pretty much nail down the guarantee that there will be exactly 2 betas, no more no less. With more betas the naming wouldn't really be as clean then (and with only 1 beta it would be odd, too).

File: docs/changelog/1_0_b1.rst (hunk 2 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_b1.rst:13]: Fair point, I'll update the wording.

**Comment by vpetrovykh** [comment]
Should the edgedb-js version tick to the next one? I mean, it's been at `0.11.0` since alpha 6 and we do mention some improvements in this changelog.

**Comment by vpetrovykh** [comment]
I've also updated the migration example to include the help the interactive tool will display for all the options.

**Review by ambv — APPROVED** [review]
